### PR TITLE
Added OrbitSsh Credentials struct

### PR DESCRIPTION
**Pull request by antonrohr**
This is a small struct that simplifies the ssh api.

(And reformats the OrbitSsh cmake file with cmake-format)

### CODE REVIEW

File: OrbitSsh/include/OrbitSsh/Credentials.h [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+namespace OrbitSsh {
+
+struct Credentials {
+  std::string host;
+  int port = -1;
+  std::string user;
+  std::filesystem::path known_hosts_path;
+  std::filesystem::path key_path;
+
+  bool NotEmpty() {
```
Comment (beckerhe) [inline OrbitSsh/include/OrbitSsh/Credentials.h:20]: const

File: OrbitSsh/include/OrbitSsh/SessionManager.h [hunk 2]
```diff
@@ -31,9 +32,7 @@ class SessionManager {
     kAuthenticated
   };
 
-  SessionManager(std::string host, int port, std::string user,
-                 std::filesystem::path known_hosts_path,
-                 std::filesystem::path key_path);
+  SessionManager(Credentials credentials);
```
Comment (beckerhe) [inline OrbitSsh/include/OrbitSsh/SessionManager.h:35]: explicit

File: OrbitSsh/include/OrbitSsh/Credentials.h (hunk 1 above)
Comment (antonrohr) [reply OrbitSsh/include/OrbitSsh/Credentials.h]: done

File: OrbitSsh/include/OrbitSsh/SessionManager.h (hunk 2 above)
Comment (antonrohr) [reply OrbitSsh/include/OrbitSsh/SessionManager.h]: done

**Pull request merged**